server/src/test/java/org/elasticsearch/index/mapper/TextFieldMapperTests.java

```diff
@@ -530,9 +530,15 @@ public void testFielddata() throws IOException {
         MapperService disabledMapper = createMapperService(fieldMapping(this::minimalMapping));
         Exception e = expectThrows(
             IllegalArgumentException.class,
-            () -> disabledMapper.fieldType("field").fielddataBuilder(FieldDataContext.noRuntimeFields("test"))
+            () -> disabledMapper.fieldType("field")
+                .fielddataBuilder(new FieldDataContext("index", null, null, MappedFieldType.FielddataOperation.SEARCH))
+        );
+        assertThat(
+            e.getMessage(),
+            containsString(
+                "Field data is disabled on [field] in [index]. Text fields are not optimised for operations that require per-document field data"
```

Comment:
checkstyle complained about this line, not sure if you've already addressed it, but we have a hard limit of 140 characters per line. Can you double check?

Reply:
My apologies for this kind of noise; I thought IntelliJ's code formatting fixes everything automatically, but I find it doesn't actually for this case. Sorry.

Reply:
no worries, happens to me as well once in a while.